draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -80,15 +80,29 @@
 	<workgroup>Internet Engineering Task Force (IETF)</workgroup>
 	<abstract>
         <t> This document describes how to extend the Internet Key Exchange Protocol
-		Version 2 (IKEv2) to allow multiple key exchanges to take place 
+	Version 2 (IKEv2) to allow multiple key exchanges to take place 
         while computing a shared secret during a Security Association (SA) setup.
-        The primary application of this feature in IKEv2 is the ability to perform one or more 
-        post-quantum key exchanges in conjunction with the classical (Elliptic Curve) Diffie-Hellman key exchange,
+	</t>
+	
+        <t> The primary application of this feature in IKEv2 is the ability to perform one or more 
+        post-quantum key exchanges in conjunction with the classical (Elliptic Curve) Diffie-Hellman (EC)DH key exchange,
         so that the resulting shared key is resistant against quantum computer attacks.
-        Another possible application is the ability to combine several key exchanges 
+	Since there is currently no post-quantum key exchange that is trusted at
+        the level that (EC)DH is trusted against conventional (non-quantum)
+        adversaries, performing multiple key exchanges with different post-quantum algorithms along
+        with the well-established classical key exchange algorithms addresses this concern, since the
+        overall security is at least as strong as each individual primitive.
```

Comment:
```        Since there is currently no post-quantum key exchange that is studied at
        the level that (EC)DH is studied, performing multiple key exchanges with different post-quantum 
        algorithms along with the well-established classical key exchange algorithms addresses this concern, since the
        overall security is at least as strong as each individual primitive.
```

Reply:
This suggestion is based on the recent message from Éric, from which I realized that the old text can be misinterpreted that current post-quantum algorithms are not fully trusted only against conventional computers. This suggestion also makes the abstract shorter.

Reply:
Yep, ok with that. Or we can even shorten it slightly:
```
... no post-quantum key exchange that is as well studied as (EC)DH, performing ...
```

Reply:
Great! Can you put this into the PR?